config/environment.js

```diff
@@ -45,7 +45,9 @@ module.exports = async function(){
     }
 
     process.env.BASE_DIR = BASE_DIR;
-    process.env.PLUGINS_DIR_PATH = path.join(BASE_DIR, 'libs', 'plugins');
+    process.env.PLUGINS_DIR_PATH = 
```

Comment:
Why we need different plugin directory for `production`?

Reply:
because `lib` directory is not being moved to `build` which is set as compiled `tsc` directory